### ISSUE TITLE
ILM unable to create write alias

### ISSUE THREAD

**Issue by robbavey** [logstash-plugins/logstash-output-elasticsearch#830]
Elasticsearch output with ILM enabled is unable to create the write alias, stopping the ILM function from working.

To reproduce:

```
output{
elasticsearch{
ilm_enabled => true # testing with all defaults first
}
````

The write alias does not created, because the escaping of the date math is not done correctly with the version of SafeURI that will be shipped with Logstash 6.6.0

The error returned from Elasticsearch is 

```
{"error":{"root_cause":[{"type":"invalid_index_name_exception","reason":"Invalid index name [%3Clogstash-%7Bnow%2Fd%7D-000001%3E], must be lowercase","index_uuid":"_na_","index":"%3Clogstash-%7Bnow%2Fd%7D-000001%3E"}],"type":"invalid_index_name_exception","reason":"Invalid index name [%3Clogstash-%7Bnow%2Fd%7D-000001%3E], must be lowercase","index_uuid":"_na_","index":"%3Clogstash-%7Bnow%2Fd%7D-000001%3E"},"status":400}
```

The issue is that this version of SafeURI is double escaping the URI, resulting in an attempt to write an alias to `http://127.0.0.1:9200/%253Clogstash-%257Bnow%252Fd%257D-000001%253E`, which fails

**Comment by robbavey** [comment]
Fixed by #831

**Comment by robbavey** [comment]
@ppf2 It will need a manual update
